dspace-api/src/main/java/org/dspace/google/GoogleAnalyticsEvent.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.google;
+
+/**
+ * This is a dataholder class for an individual event to be sent to Google Analaytics
+ *
+ * @author April Herron
+ */
+public class GoogleAnalyticsEvent {
+
+    private String cid;
+    private String uip;
+    private String ua;
+    private String dr;
+    private String dp;
+    private String dt;
+    private long time;
+
+    GoogleAnalyticsEvent(String cid, String uip, String ua, String dr, String dp, String dt, long time) {
+        this.cid = cid;                      // Client ID
+        this.uip = uip;                      // User IP
+        this.ua = ua;                        // User Agent
+        this.dr = dr;                        // Document Referrer
+        this.dp = dp;                        // Document Path
+        this.dt = dt;                        // Document Title
+        this.time = time;                    // Time of event
+    }
+
+    public String getCid() {
```

Comment:
Could we add JavaDocs to each of these `get` methods and move the description to the methods?  I see you noted on line 26 that "cid" = "Client ID".  But, I think those same notes would be useful to add as JavaDocs on the getters in this class...it just makes it a lot easier to understand this class without having to constantly scroll to constructor to translate each abbreviation.